File: Week 1/Day 1/algo-fibonacci/code.js
function findFibonacci(num) {
  if (num < 2) {
    return num;
  }

  return findFibonacci(num - 1) + findFibonacci(num - 2);
}

console.log(findFibonacci(0) === 0);
console.log(findFibonacci(2) === 1);
console.log(findFibonacci(5) === 5);
console.log(findFibonacci(8) === 21);
console.log(findFibonacci(11) === 89);
console.log(findFibonacci(14) === 377);
console.log(findFibonacci(17) === 1597);
console.log(findFibonacci(20) === 6765);

File: Week 1/algo-99-bottles/code.py
def bottles(num):
    bottlesLeft = num
    start = num
    while bottlesLeft != 0:
        if (bottles == 1):
            print(
                f"{bottlesLeft} bottle of beer on the wall, {bottlesLeft} bottle of beer.")
        else:
            print(
                f"{bottlesLeft} bottles of beer on the wall, {bottlesLeft} bottles of beer.")

        bottlesLeft -= 1

        if (bottlesLeft == 0):
            print(
                "Take one down and pass it around, no more bottles of beer on the wall.")
            print("No more bottles of beer on the wall, no more bottles of beer.")
            print(
                f"Go to the store and buy some more, {start} bottles of beer on the wall.")
        else:
            print(
                f"Take one down and pass it around, {bottlesLeft} bottles of beer on the wall.")


bottles(99)
